receiver/skywalkingreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,31 @@
+receivers:
+#  otlp:
```

Comment:
Is this commented out receiver intended? I think it can be simply removed

Reply:
ok